scipy/stats/stats.py

```diff
@@ -4275,6 +4277,158 @@ def mannwhitneyu(x, y, use_continuity=True):
     return MannwhitneyuResult(smallu, distributions.norm.sf(z))
 
 
+# TODO: add paired=False, do a signed-rank test if paired=True or y is not provided. See morestats.wilcoxon
+def mww(x, y, correction=True, exact='auto',
+             alternative='two-sided'):
+    """
+    Computes two-sample unpaired Mann-Whitney-Wilcoxon tests.
+
+    Parameters
+    ----------
+    x : array_like
+        The first set of measurements.
+    y : array_like
+        The second set of measurements.
+    correction : bool, optional
+        If True, apply continuity correction by adjusting the Wilcoxon rank
+        statistic by 0.5 towards the mean value when computing the z-statistic.
+        Default is False.
+    exact : bool, optional
+        Whether an exact test should be performed on the data. See notes for
+        default behavior.
```

Comment:
I agree. The user arguments input here should be boolean--but the default value was changed to 'auto' instead of None for intuitiveness of function ([see above](https://github.com/scipy/scipy/pull/4933#discussion_r31585248)). Perhaps this is a good time to have a broader discussion about SciPy convention and style regarding this issue? @rgommers @phobson